.github/ISSUE_TEMPLATE/feature-request.yml

```
@@ -0,0 +1,41 @@
+name: 🚀 | Feature request
+description: >
+  Request a new feature
+title: "[🚀 | Feature request]: "
+labels:
+  - triage
+body:
+  - type: dropdown
+    id: related-to-problem
+    attributes:
+      label: Is your feature request related to a problem?
+      multiple: false
+      options:
+        - "Yes"
+        - "No"
+    validations:
+      required: true
+  - type: textarea
+    id: problem-description
+    attributes:
+      label: If your feature request is related to a problem describe it
+      description: |
+        Describe the problem you are trying to solve with this feature request.
+  - type: textarea
+    id: feature-description
+    attributes:
+      label: Describe the solution you would like
+      description: |
+        Please describe the behavior, include possible design draft,
+        how the users would use it, why it is important for you.
+    validations:
+      required: true
+  - type: textarea
+    id: considered-alternative
+    attributes:
+      label: Describe alternatives you have considered
+      description: |
+        Please describe alternative solutions or features you have considered,
+        why this feature instead of another one ?
+    validations:
+      required: true
```

Better leave optional? If alternative solutions are not applicable people might end putting random text just to validate. 
```suggestion
      required: false
```